library/Zend/Code/Generator/ValueGenerator.php

```diff
@@ -239,7 +318,7 @@ public function generate()
                 );
                 foreach ($rii as $curKey => $curValue) {
                     if (!$curValue instanceof ValueGenerator) {
-                        $curValue = new self($curValue);
+                        $curValue = new self($curValue, self::TYPE_AUTO, self::OUTPUT_MULTIPLE_LINE, $this->getConstants());
```

Comment:
So native `ArrayObject` is suitable for this case and `Stdlib` version is not required, isn't it?